src/crypto-api.ts

```diff
@@ -185,6 +185,17 @@ export interface CryptoApi {
      * @returns True if secret storage is ready to be used on this device
      */
     isSecretStorageReady(): Promise<boolean>;
+
+    /**
+     * Get the status of our cross-signing keys.
+     *
+     * - if there is a public key on this device
+     * - if the private master, userSigning and selfSigning keys are store in the SecretStorage
+     * - if the private master, userSigning and selfSigning keys are cached locally
+     *
+     * @returns the current status of cross-signing keys
```

Comment:
Look at https://pr3452--js-sdk-docs-previews.netlify.app/stable/interfaces/crypto.cryptoapi#getCrossSigningStatus: this should be a proper sentence, with a capital letter and a full stop.

```suggestion
     * @returns The current status of cross-signing keys: whether we have public and private keys cached locally, and whether the private keys are in secret storage.
```